project/Dependencies.scala

```diff
@@ -45,8 +45,10 @@ object Dependencies {
   val scala213Version = "2.13.8"
   // To get the fix for https://github.com/lampepfl/dotty/issues/13106
   // and restored static forwarders
-  val scala3Version = "3.1.2"
-  val allScalaVersions = Seq(scala213Version, scala212Version, scala3Version)
+  val scala31Version = "3.1.3"
+  val scala32Version = "3.2.0"
```

Comment:
Thanks for trying this @He-Pin 

What I have heard from @SethTisue is that we can build with Scala 3.1.x and that will work fine for libraries/applications that are using 3.2.x. Scala 3 minor versions are backwards compatible.

Therefore I suggest that we don't cross build for Scala 3.2.

Reply:
I try to use `+ 3.x` and `++ 3.x` both not work now .
https://github.com/sbt/sbt/issues/7018

Reply:
@patriknw I try this to address find the issue of dotty 3.2 https://github.com/lampepfl/dotty/issues/16018.
And yes, compiles with 3.1 can consumed by 3.2.

Reply:
agree, it's only necessary to build for one Scala 3 version